servers/src/main/java/tachyon/master/rawtable/RawTableMaster.java

```diff
@@ -138,7 +138,10 @@ public long createRawTable(TachyonURI path, int columns, ByteBuffer metadata)
 
     // Create a directory at path to hold the columns
     MkdirOptions options =
-        new MkdirOptions.Builder(MasterContext.getConf()).setRecursive(true).build();
+        new MkdirOptions.Builder(MasterContext.getConf())
+            .setPersisted(true)
```

Comment:
In original code it is never true. But by setting it to true here, we don't have to create the directories on client-side.